runtime/jvmMain/src/kotlinx/benchmark/jvm/JvmBenchmarkRunner.kt

```diff
@@ -56,6 +56,15 @@ fun main(args: Array<String>) {
         }
     }
 
+    val profilerName = config.advanced["jvmProfiler"]
+    when (profilerName) {
+        "gc" -> jmhOptions.addProfiler("gc")
+        "stack" -> jmhOptions.addProfiler("stack")
+        "cl" -> jmhOptions.addProfiler("cl")
+        "comp" -> jmhOptions.addProfiler("comp")
+        else -> throw IllegalArgumentException("Invalid value for 'jvmProfiler': $profilerName. Accepted values: gc, stack, cl, comp")
```

Comment:
What if `jvmProfiler` is not provided at all?

Reply:
Added a null case to address this https://github.com/Kotlin/kotlinx-benchmark/pull/146/commits/452c353fa30d886d5e65bbca170e7a568143a6ff.